lib/Echidna/Output/Corpus.hs

```diff
@@ -0,0 +1,47 @@
+module Echidna.Output.Corpus where
+
+import Prelude hiding (Word)
+
+import Control.Monad (unless)
+import Control.Monad.Catch (bracket)
+import Data.Aeson (ToJSON(..), decodeStrict, encodeFile)
+import Data.Hashable (hash)
+import Data.Maybe (catMaybes)
+import System.Directory (createDirectoryIfMissing)
+
+import qualified System.Directory as SD
+import qualified Data.ByteString as BS
+
+import Echidna.Types.Tx
+
+listDirectory :: FilePath -> IO [FilePath]
```

Comment:
maybe move these directory functions to a util module

Reply:
Fixed.